keyboards/handwired/sauceball/keymaps/stock/keymap.c

```diff
@@ -0,0 +1,127 @@
+// Copyright 2023 xia0 (@xia0)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+#include "print.h"
+#include "mousekey.h"
```

Comment:
```suggestion
```

Reply:
thank you :') #21897 is merged into develop branch. Does that mean I'll need to wait until its merged into master before this PR will be?